build/print.go

```diff
@@ -160,7 +161,18 @@ func (p *printer) file(f *File) {
 	p.newlineIfNeeded()
 }
 
-func (p *printer) statements(stmts []Expr) {
+func (p *printer) statements(stmts []Expr, isTopLevel bool) {
```

Comment:
How is `isTopLevel` different from `p.level == 0`?
I think you can remove the argument or rename it.

Reply:
`p.level` is changed inside this function, so it's 0 when the statements on the first level are printed.

Reply:
I still don't understand the difference between `isTopLevel` and `p.level == 0`.
Can you make clear what is `isTopLevel`?

Reply:
`isTopLevel` is true if the top statements are being printed (i.e. directly in the File node).

If statements from a, for example, def block are being printed, `p.level` will be increased by 1, but only inside the function, so it's still 0 when the function is called (given that the def-block itself is a top-level statement).

Reply:
Do you mind clarifying the code?

Reply:
Do you have any suggestions?

I can split the function in two, `statements(stmts []Expr)` (to print just a sequence of statements) and `nestedStatements(stmts []Expr)` (to increase the indentation and level and then call `statements`), does it look better?

Reply:
Yes, it's probably clearer than a boolean argument

Reply:
Done.